sdk/keyvault/Azure.Security.KeyVault.Keys/CHANGELOG.md

```diff
@@ -12,12 +12,19 @@ Thank you to our developer community members who helped to make the Key Vault cl
 
 ### Breaking Changes
 
+> [!NOTE]  
```

Comment:
There are no breaking changes in a beta, because there is no stability guarantee.   Please don't list beta => beta or beta => stable changes as breaking.

Reply:
Yeah.   Stable release notes should concentrate on the delta between stable releases.   Its fine to say something like "Includes all of the features from Beta.3" if we want to be lazy or call them out specifically to make it easier to read for people.